pallets/Polkadex_Ido/src/lib.rs

```diff
@@ -342,16 +342,14 @@ decl_module! {
             let max_amount = <WhiteListInvestors<T>>::get(round_id, investor_address.clone());
             ensure!(amount >= max_amount, Error::<T>::NotAValidAmount);
             ensure!(<InfoInvestor<T>>::contains_key(&investor_address), <Error<T>>::InvestorDoesNotExist);
-            <T as Config>::Currency::transfer(AssetId::POLKADEX, &investor_address, &Self::get_wallet_account(), amount)?;
-            let current_block_no = <frame_system::Pallet<T>>::block_number();
             let funding_round = <InfoFundingRound<T>>::get(round_id);
-            ensure!(current_block_no < funding_round.close_round_block && current_block_no > funding_round.start_block, <Error<T>>::NotAllowed);
```

Comment:
Why was this check removed? As I see it, without this it is possible to participate_in_round outside of the allocated time slot for this round.

Reply:
@mambisi @simonsso

Reply:
It is added here https://github.com/Polkadex-Substrate/Polkadex/pull/279/files#diff-a1733927d0808b2a6f1042df597fd17f8eea6c273fcb1479aa60306ef8e916a0R388